selfdrive/ui/ui.cc

```diff
@@ -357,11 +357,17 @@ void handle_message(UIState *s, Message * msg) {
     struct cereal_RadarState datad;
     cereal_read_RadarState(&datad, eventd.radarState);
     struct cereal_RadarState_LeadData leaddatad;
+    struct cereal_RadarState_LeadData leaddatae;
     cereal_read_RadarState_LeadData(&leaddatad, datad.leadOne);
+    cereal_read_RadarState_LeadData(&leaddatae, datad.leadTwo);
```

Comment:
Can you reuse the same struct? Just do the `cereal_read_RadarState_LeadData(&leaddatae, datad.leadTwo);` after you're done with the first lead.

Reply:
Done. Even though you have a spelling mistake in your comment